File: app/src/main/java/com/cjztest/gldrawline/MainActivity.java
package com.cjztest.gldrawline;
import android.app.Activity;
import android.content.pm.ActivityInfo;
import android.os.Bundle;
import android.view.Window;
import android.view.WindowManager;

public class MainActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) 
    {
        super.onCreate(savedInstanceState);
		setContentView(new PanelView(this));

    }

    @Override
    protected void onResume() {
        super.onResume();
    }

    @Override
    protected void onPause() {
        super.onPause();
    }    
}

File: app/src/main/java/com/physicEffect/threeDWave/MainActivity.kt
package com.physicEffect.threeDWave

import android.app.Activity
import android.os.Bundle

class MainActivity: Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(ThreeDimensionWaveGLAnimate(this))
    }
}